### ISSUE TITLE
Error 500 after upgrade – Smarty 'Autoloader.php' failed to open

### ISSUE THREAD

**Issue by phavel** [eventum/eventum#233]
After upgrade 3.1.5→3.1.7 HTTP ERROR 500 everywhere.

In PHP error log is:

```
[29-Dec-2016 21:26:26 Europe/Prague] PHP Warning:  require(/home/users/…/…/…/…/vendor/smarty/smarty/libs/Autoloader.php): failed to open stream: No such file or directory in /home/users/…/…/…/…/vendor/smarty/smarty/libs/bootstrap.php on line 15

[29-Dec-2016 21:26:26 Europe/Prague] PHP Fatal error:  require(): Failed opening required '/home/users/…/…/…/…/vendor/smarty/smarty/libs/Autoloader.php' (include_path='/home/users/…/…/…/…/vendor/pear/pear_exception:/home/users/…/…/…/…/vendor/pear/auth_sasl:/home/users/…/…/…/…/vendor/pear/pear-core-minimal/src:/home/users/…/…/…/…/vendor/pear/db:/home/users/…/…/…/…/vendor/pear/mail:/home/users/…/…/…/…/vendor/pear-pear.php.net/Mail_Mime:/home/users/…/…/…/…/vendor/pear-pear.php.net/Mail_mimeDecode:/home/users/…/…/…/…/vendor/pear-pear.php.net/Math_Stats:/home/users/…/…/…/…/vendor/pear-pear.php.net/Net_LDAP2:/home/users/…/…/…/…/vendor/pear-pear.php.net/Net_Socket:/home/ in /home/users/…/…/…/…/vendor/smarty/smarty/libs/bootstrap.php on line 15 
```

**Comment by glensc** [comment]
fixed in 0789211

**Comment by glensc** [comment]
3.1.9 released
